src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private int size;
+    private int threshold;
+    private List<Entry<K, V>>[] buckets;
+
+    public MyHashMap() {
+        this(INITIAL_CAPACITY);
+    }
+
+    public MyHashMap(int capacity) {
+        this.buckets = new List[capacity];
+        for (int i = 0; i < capacity; i++) {
+            buckets[i] = new ArrayList<>();
+        }
```

Comment:
Using raw types is not recommended. You should specify the type parameters when creating new instances of generic types. Use the diamond operator to avoid type safety issues: `new ArrayList<>()`.